.github/workflows/build-packages.yml

```diff
@@ -123,7 +123,8 @@ jobs:
       run: |
         cd jans/
         sed -i "s/%VERSION%/${{ steps.previoustag.outputs.version }}/g" checksum.sh
-        sudo ./checksum.sh      
+        sudo ./checksum.sh
+        ls /home/runner/work/jans/jans/jans/rpmbuild/RPMS/x86_64/
```

Comment:
What is the purpose of running `ls` command here? Was it for debugging the contents of the directory?